### ISSUE TITLE
Convert `tools/alpine` to `linuxkit pkg`

### ISSUE THREAD

**Issue by ijc** [linuxkit/linuxkit#2586]
By inspection of `tools/alpine/Makefile` there seem to be two main issues:

The main issue seems to be the use of different algorithm for calculating the hash -- it is taking a hash of the packages installed and various other files instead of using the tree-sh, which also means the image must be built and then tagged (with treesh hash it can be precalculated and therefore done first).

Fixing this directly would involve making the hash algorithm somehow configurable. Or adding `--no-tag` and `--iidfile` options to `linuxkit pkg build` to allow a `Makefile` to drive the process more manually.

Secondly there is some complexity because the arm64 image is not signed. We should fix that.

**Comment by justincormack** [comment]
There should be upstream signed multiarch Alpine base images soon.

**Comment by justincormack** [comment]
See https://github.com/gliderlabs/docker-alpine/pull/337 for progress on multiarch ALpine

**Comment by ijc** [comment]
Do they help? If we were to switch to those we'd need to enable networking for almost all package builds, plus we would lose repeatability for the builds.

Trade-off of not having to mess around with updating the mirror feels superficially compelling right now though.

**Comment by justincormack** [comment]
No, just for this build, to resolve the non signedness issue, and simplify by removing the build args for base package

**Comment by rn** [comment]
https://github.com/linuxkit/linuxkit/pull/2653 converted `tools/alpine` to use the alpine multi-arch image, so that's the minor of the two issue above fixed.
